src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,13 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery[] balls = new Lottery[3];
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = new Lottery();
```

Comment:
You are creating a new instance of `Lottery` in each iteration of the loop. According to the checklist, it's better to create a single instance of the `Lottery` class and reuse it to generate the balls. This will optimize your code and adhere to best practices .